File: Testing/Code for Testing/Early Basic Syntax Testing/sysfuncs/test_sysfuncs.py
from dataclasses import dataclass, field
import random
import time

def _ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class _ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)


def main():
    t = int(time.time())
    if (t > 0):
        print('time ok')
    c = time.time_ns()
    if (c > 0):
        print('clock ok')
    random.seed(42)
    r = random.random()
    if (r >= 0.0):
        if (r <= 1.0):
            print('random ok')
    ri = random.randint(1, 10)
    if (ri >= 1):
        if (ri <= 10):
            print('random_int ok')
    return 0


if __name__ == "__main__":
    main()

File: Testing/Code for Testing/Early Basic Syntax Testing/sysfuncs/test_sysfuncs.js

function main() {
    let t = Math.floor(Date.now() / 1000);
    if ((t > 0)) {
        console.log("time ok");
    }
    let c = (typeof process !== 'undefined' ? Number(process.hrtime.bigint()) : Math.floor(performance.now() * 1e6));
    if ((c > 0)) {
        console.log("clock ok");
    }
    undefined /* random_seed not supported in JS */;
    let r = Math.random();
    if ((r >= 0.0)) {
        if ((r <= 1.0)) {
            console.log("random ok");
        }
    }
    let ri = (Math.floor(Math.random() * (10 - 1 + 1)) + 1);
    if ((ri >= 1)) {
        if ((ri <= 10)) {
            console.log("random_int ok");
        }
    }
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();
